client-code-gen/gen/model/fam-forest-client-status.ts

```diff
@@ -0,0 +1,39 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Forest Access Management - FAM - API
+ *  Forest Access Management API used by the Forest Access Management application to Define who has access to what apps, and what roles they will operate under  once access is granted. 
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: guy.lafleur@montreal.canadians.ca
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+// May contain unused imports in some cases
+// @ts-ignore
+import { FamForestClientStatusType } from './fam-forest-client-status-type';
+
+/**
+ * 
+ * @export
+ * @interface FamForestClientStatus
+ */
+export interface FamForestClientStatus {
+    /**
+     * 
+     * @type {FamForestClientStatusType}
+     * @memberof FamForestClientStatus
+     */
+    'status_code': FamForestClientStatusType;
+    /**
+     * 
+     * @type {string}
+     * @memberof FamForestClientStatus
+     */
+    'description': string;
```

Comment:
just wondering shall we give the description a real description? or it doesn't matter?